modules/nw-metallb-operator-initial-config.adoc

```diff
@@ -18,7 +22,7 @@ After you install the Operator, you need to configure a single instance of a Met
 [source,terminal]
 ----
 $ cat << EOF | oc apply -f -
```

Comment:
@mikemckiernan Most of it looks good. I just have a thought of not including prompt sign '$' where ever there is cat << EOF command to allows user to use the clipboard functionality to copy and paste the YAML.

Reply:
Thanks @asood-rh, including the prompt is the Red Hat docs convention, but you've uncovered some kind of browser-specific bug that causes the prompt to get included in the paste.  I'll investigate that with my team.